pkg/object/meshcontroller/service/service.go

```diff
@@ -52,15 +52,15 @@ func New(superSpec *supervisor.Spec) *Service {
 	return s
 }
 
-// Lock locks all store, it will do cluster panic if failed.
+// Lock - locks all store, it will do cluster panic if failed.
```

Comment:
why add a `-`?

Reply:
golint requires the comment must be lead by function name. because Lock and Unlock is a pair function. And the `-` just separate the function name and the comments. otherwise, it seems a bit stuttering.

Reply:
The official go source code for `Mutex.Lock` & `Mutex.Unlock` doesn't have it.

```go
// Lock locks m.
// If the lock is already in use, the calling goroutine
// blocks until the mutex is available.
func (m *Mutex) Lock() {


// Unlock unlocks m.
// It is a run-time error if m is not locked on entry to Unlock.
//
// A locked Mutex is not associated with a particular goroutine.
// It is allowed for one goroutine to lock a Mutex and then
// arrange for another goroutine to unlock it.
func (m *Mutex) Unlock() {

```

Reply:
```suggestion
// Lock locks all store, it will do cluster panic if failed.
```